lib/install/validate/validator.go

```diff
@@ -902,3 +914,28 @@ func (v *Validator) syslog(op trace.Operation, conf *config.VirtualContainerHost
 		RAddr:   host,
 	}
 }
+
+// FIXME ATC DEBT setting this value needs to be moved to Dispatcher
+// https://github.com/vmware/vic/issues/6803
+// set PersistNetworkBacking key to "true"
+func (v *Validator) ConfigureVCenter(ctx context.Context) error {
+	op := trace.FromContext(ctx, "Set vCenter serial port backing")
+	defer trace.End(trace.Begin("", op))
+
+	errMsg := "Set vCenter settings SKIPPED"
+	if !v.sessionValid(op, errMsg) {
+		return nil
+	}
+	if !v.IsVC() {
```

Comment:
Minor: can this check be moved above the session valid check? If the target isn't vCenter we can exit quickly that way.

Reply:
If the session isn't valid I don't think we can get that information.